### ISSUE TITLE
GDScript 2.0: Documentation comments do not appear in inspector tooltips

### ISSUE THREAD

**Issue by h0lley** [godotengine/godot#59270]
### Godot version

v4.0.alpha4.official [f47097973]

### System information

Ubuntu, Nvidia, Vulkan

### Issue description

Looking at the merged PR https://github.com/godotengine/godot/pull/41095, inspector tooltips for `@export` variables should already be implemented. However, I can't get them to show up. They do appear in the documentation (F1) just fine, but the tooltip text remains "No description".

![image](https://user-images.githubusercontent.com/23386940/158979506-5f948c19-6661-4c94-97ad-d92719509c08.png)

A restart of the editor does not help.

### Steps to reproduce

Code used in above example:
```swift
extends Node
class_name MyClass

## My documentation for my_inspector_setting
@export var my_inspector_setting: int
```
Or get the minimal reproduction project,
1. Press F1 and check the docs of MyClass, notice the description of `my_inspector_setting`
2. Open `main.tscn` and hover the property in the inspector, notice "No description"

### Minimal reproduction project

[Godot 4 Inspector tooltip.zip](https://github.com/godotengine/godot/files/8303412/Godot.4.Inspector.tooltip.zip)

**Comment by FaerDerr** [comment]
What's up with this feature ? 
This has been removed from the 4.0 milestone, is it expected to be in 3.5 or is it just stranded ?

**Comment by akien-mga** [comment]
It's not critical to solve before releasing 4.0, so in such case we don't put a milestone, which just means it will be fixed when it's fixed. Would still be nice to have fixed in 4.0 of course if anyone wants to work on it.

**Comment by FaerDerr** [comment]
Original proposal has been around since 2016 ( #4370 )
Couldn't we do something to speed up the process for 4.0 ? Maybe asking directly in discord or something, It seems like a long-requested update

**Comment by akien-mga** [comment]
That proposal has been implemented already, and works fine. What's missing is literally just hooking it up in tooltips, which can be done by any motivated contributor.

**Comment by h0lley** [comment]
in opening post I mentioned that this should already be implemented judging by the screenshot in #41095 so I thought let's bisect it, but I just compiled 42bfa16 and was still unable to get tooltips, so perhaps the screenshot was just a concept.
@ThakeeNathees perhaps you could clarify. have tooltips been implemented at some point?
